webapp/storybook/layout.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <ds-container class="container">
+  <div class="container">
     <slot />
-  </ds-container>
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-04-07T13:58:56Z" title="Tuesday, April 7th 2020, 3:58:56 pm +02:00">Apr 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,7 +1,7 @@
 <template>
-  <ds-container class="container">
+  <div class="container">
     <slot />
-  </ds-container>
```


nice, thank you!

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-16T14:48:11Z" title="Thursday, April 16th 2020, 4:48:11 pm +02:00">Apr 16, 2020</time>_

---


I'll make it a habit :)